src/main/java/com/example/Car.java

```diff
@@ -0,0 +1,12 @@
+package com.example;
+
+public class Car {
+            private int speed;
+
+
+    public int getSpeed() {return speed;
+    }
+
+    public void setSpeed(int speed) {this.speed = speed;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.blocks.LeftCurlyCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' at column 37 should have line break after.